dev-requirements.txt

```diff
@@ -3,3 +3,10 @@ twine>=1.11.0
 wheel>=0.31.0
 black
 flake8
+notebook
+git+https://github.com/hgrecco/pint.git
```

Comment:
pint is not available via pypi?

Reply:
I think this was just to be able to install latest version with Pandas support